_posts/2020-09-23-cytech-customer-story.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: 'Cytech uses Quarkus to save Greek Tourism'
+date: 2020-09-23
+tags: user-story
+synopsis: Cytech uses Quarkus to save Greek Tourism
+author: dandreadis
+---
+
+:imagesdir: /assets/images/posts/quarkus-user-stories/cytech
+
+= Cytech uses Quarkus to save Greek Tourism
+
+image::cytechLogoLarge.png[]
+
+== Tourism and Covid-19
+It's not new that the tourism industry is one the most affected industries from the Covid-19 pandemic. Hotels, restaurants, airlines, cruises are some of the businesses that had a really tough time this summer. Many countries, especially those whose economies rely on tourism, were looking for solutions in order to open up the borders for tourists while not  imposing their citizens to a great health risk. There are no magic solutions here but there is always room for doing your best to satisfy both requirements.
+
+The Greek government decided at some point to open up the borders in order to help the economy but it would be too reckless to do it without precautions and measures. That’s why they decided to build a system based on advanced technology that would choose the right travelers for Covid-19 testing. The idea is the following: if you cannot afford to test everyone then you should test those who are more likely to be positive to a Covid-19 test. This way the Greek state can use their resources wisely and at the same time save money for tests.
```

Comment:
```suggestion
The Greek government decided at some point to open up the borders to help the economy, but it would be too reckless to do it without precautions and measures. That's why they decided to build a system based on advanced technology that would choose the right travelers for Covid-19 testing. The idea is the following: if you cannot afford to test everyone, you should test those who are more likely to be positive to a Covid-19 test. This way, the Greek state can use its resources wisely and, at the same time, save money for tests.
```